Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:3.13-alpine3.21
+LABEL maintainer="dmysamo@gmail.com"
+
+ENV PYTHOUNNBUFFERED 1
+
+WORKDIR app/
```

Comment:
The `WORKDIR` should be specified as an absolute path. Consider changing it to `/app` instead of `app/`.